internal/eventstore/models/version.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import (
+	"regexp"
+
+	"github.com/caos/zitadel/internal/errors"
+)
+
+var versionRegexp = regexp.MustCompile(`^v[0-9]+(\.[0-9]+){0,2}$`)
+
+type Version string
```

Comment:
just for my understanding, where do we get this version from?

Reply:
It's set by the service which wants to push an aggregate (internal/org-pkg for example)